tests/test_routing.py

```diff
@@ -19,6 +19,16 @@ def user(request):
     return Response(content, media_type="text/plain")
 
 
+def user_me(request):
+    content = "User fixed me"
+    return Response(content, media_type="text/plain")
+
+
+def user_no_match(request):
```

Comment:
Tests are failing since these two lines are never reached by coverage.

I'd suggest: `raise NotImplementedError()  # pragma: no cover`

Reply:
Ah I see you're already ahead of me! :)

Reply:
Hehe :smile: 

Still, do you think it's more explicit / better to use a `raise NotImplementedError()` ?

Reply:
I'd marginally prefer it, but not *that* fussed either way.